component/otelcol/config_debug_metrics.go

```diff
@@ -0,0 +1,16 @@
+package otelcol
+
+// DebugMetricsArguments configures internal metrics of the components
+type DebugMetricsArguments struct {
+	DisableHighCardinalityMetrics bool `river:"disable_high_cardinality_metrics,attr,optional"`
```

Comment:
TBH I'm not sure if we should just call this `high_cardinality_metrics`. IMO we can merge this as it is for now, and I'll have a discussion with the other people on the team if they'd be in favour of changing it prior to the next release.